config/v1/types_node.go

```diff
@@ -107,76 +107,36 @@ type WorkerLatencyProfileStatus struct {
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	// +optional
-	Conditions []WorkerLatencyStatusCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 }
 
-// WorkerLatencyStatusConditionType is an aspect of WorkerLatencyProfile state.
-type WorkerLatencyStatusConditionType string
-
+// Different condition types to be used by MachineConfigOperator, Kube Controller
+// Manager Operator and Kube API Server Operator
 const (
 	// Progressing indicates that the updates to component (Kubelet or Controller
 	// Manager or Kube API Server) is actively rolling out, propagating changes to the
 	// respective arguments.
-	WorkerLatencyProfileProgressing WorkerLatencyStatusConditionType = "Progressing"
+	MachineConfigProgressing         = "MachineConfigProgressing"
+	KubeControllerManagerProgressing = "KubeControllerManagerProgressing"
+	KubeAPIServerProgressing         = "KubeAPIServerProgressing"
 
 	// Complete indicates whether the component (Kubelet or Controller Manager or Kube API Server)
 	// is successfully updated the respective arguments.
-	WorkerLatencyProfileComplete WorkerLatencyStatusConditionType = "Complete"
+	MachineConfigComplete         = "MachineConfigComplete"
```

Comment:
shouldn't this be a Kubelet?

Reply:
cc: @sairameshv 
Which one would be more appropriate?
`MachineConfigComplete` or `KubeletComplete`? / the alike.

@atiratree the way MCO is actually handling Kubelet args is via Machine Configs. But on the other hand, if we wanna stick to operand naming "kubelet" is more apt.

Reply:
@atiratree used "Kubelet..." in 3ffa44f.

Reply:
Yeah @swghosh , "kubelet" sounds good.